src/test/java/com/wikia/webdriver/testcases/mercurytests/VerbatimTests.java

```diff
@@ -0,0 +1,43 @@
+package com.wikia.webdriver.testcases.mercurytests;
```

Comment:
move it to package like widgets or thirdparties. Verbatim is mediawiki syntax relatex term, not functionality